CHANGELOG.md

```diff
@@ -6,15 +6,19 @@
 
 ## [[UnreleasedUniFFIVersion]] (backend crates: [[UnreleasedBackendVersion]]) - (_[[ReleaseDate]]_)
 
-### What's fixed?
+- The async runtime can be specified for constructors/methods, this will override the runtime specified at the impl block level.
```

Comment:
```suggestion
### What's changed?
- The async runtime can be specified for constructors/methods, this will override the runtime specified at the impl block level.
```